### ISSUE TITLE
Improve Page Load Time

### ISSUE THREAD

**Issue by nickpaventi** [nickpaventi/culligan-diy#143]
Wanted to keep this on the radar before Go Live:

Reduce # of  HTTP requests
Minify the JS and CSS
Add caching
Add image dimensions (height and width) to img elements

**Comment by MartyBeckers** [comment]
Yes I put as many of the changes in place as possible, although they haven't made it up to the staging server yet (http://stg.culligandiy.com) and that's where the testing should really take place because just the act of developing slows the site way down.  For instance, if you've ever attempted to bring up a page and experienced a long delay of between 30 seconds and 5 minutes, that's because I've made a change to the custom .Net class.  Whenever I press that save button, Kentico automatically clears the cache and then each time a page is loaded for the first time, it goes through and rebuilds everything with .Net logic.  Then after it finishes loading, a refresh of that same page loads instantly but the next page you visit will need to rebuild again.  Once everything makes it up to staging/live where no development is taking place, the tests should reflect all of the performance changes.
